morse.rb

```diff
@@ -0,0 +1,62 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z',
+  '.----' => '1',
+  '..---' => '2',
+  '...--' => '3',
+  '....-' => '4',
+  '.....' => '5',
+  '-....' => '6',
+  '--...' => '7',
+  '---..' => '8',
+  '----.' => '9',
+  '-----' => '0',
+  '--..--' => ',',
+  '.-.-.-' => '.',
+  '..--..' => '?',
+  '-..-.' => '/',
+  '-....-' => '-',
+  '-.--.' => '(',
+  '-.--.-' => ')',
+  '.--.-.' => '@',
+  '-...-' => '=',
+  '.-.-.' => '+'
+}.freeze
+
+def decode_char(char)
+  MORSE_CODE[char]
+end
+
+def decode_word(word)
+  word.split.map { |char| MORSE_CODE[char] || char }.join(' ')
+end
+
```

Comment:
- [OPTIONAL] Great work creating a method that translates a **_morse code character_** into a letter. However, based on requirements, the method `decode_char` should be used in the `decode_word` method. This would make your code _**DRY**_ and non-repetitive. I would advise you to implement it. 👍 😎 

```ruby
def decode_word(word)
  word.split(' ').map { |char| decode_char(char) }.join('')
end
```

Reply:
Thank you for this suggestion. We have taken it into consideration and we will implement it right away @Gambit142